app/templates/events/event_list.html

```diff
@@ -13,6 +13,7 @@
   <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
 {% endblock %}
 {% block app_content %}
+<main id="maincontent"></main>
```

Comment:
Let's add this id to a more global location, like an already existing element that surrounds our app content. (as we discussed, potentially line 54 in base.html)

Reply:
Done.